### PR TITLE
Replace ethereumjs-util with @ethereumjs/util

### LINKED ISSUE #6021: There are dependencies to "ethereumjs-util" and "@ethereumjs/util"
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Internally the code refers to `ethereumjs-util` but due to also using `@ethereumjs/tx` and `@ehterumjs/common` but those modules referring to `ethereumjs-util` by it's new name `@ethereumjs/util` there is some code duplication inside of packaged output which also makes it harder to properly upgrade to newer versions of crypto modules.

### Expected Behavior

Ideally `ethereumjs-util` should no longer be used if the new naming is used for the other modules.

### Steps to Reproduce

```js
import "web3-eth-accounts"

// will cause both modules to be in the bundle because web3-eth-accounts requires `ethereumjs-util` and `@ethereumjs/tx` and 
// `@ethereumjs/common` which in turn include `@ethereumjs/util`
```

### Web3.js Version

1.9.0

### Environment

- Operating System: Ubuntu 22.10
- Browser: Chrome Version 111.0.5563.149 (Official Build) (64-bit)
- Node.js Version: v18.14.0
- NPM Version: yarn 3.3.0 npm 9.3.1
- Webpack: 5.79.0

### Anything Else?

The current way to prevent upgrade problems is to leave
"ethereum-cryptography": "1.2.0",
and
"@noble/secp256k1": "1.7.1",

### PR DESCRIPTION
**Pull request by avkos**
## Description

Fixes #6021

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,6 +106,7 @@
         "bignumber.js": "^9.0.1",
         "bn.js": "^5.2.1",
         "browserify": "^16.5.2",
+        "browserify-zlib": "^0.2.0",
```
Comment (jdevcs) [inline package.json:109]: why this lib is added?

**Review by luu-alex — APPROVED** [review]

**Comment by luu-alex** [comment]
other than junaids comment lgtm

File: package.json (hunk 1 above)
Comment (avkos) [reply package.json:109]: we need this Polyfill otherwise build fails by [error](https://github.com/web3/web3.js/actions/runs/5548744411/jobs/10132070027)

**Review by jdevcs — APPROVED** [review]

**Review by Muhammad-Altabba — APPROVED** [review]

File: test/e2e.method.signing.js [hunk 2]
```diff
@@ -539,6 +539,7 @@ describe('transaction and message signing [ @E2E ]', function() {
     });
 
     it('eth.accounts.sign', async function(){
+        this.timeout(30000);
```
Comment (Spacesai1or) [inline test/e2e.method.signing.js]: Seems weird to require such a long timeout for a signing call

**Comment by jdevcs** [comment]
@avkos , before merging tests should be passing:
` Error: Returned error: The method personal_sign does not exist/is not available`

**Pull request merged**